auth0/src/main/java/com/auth0/android/provider/WebAuthProvider.java

```diff
@@ -155,7 +152,18 @@ public Builder useFullscreen(boolean useFullscreen) {
          * @return the current builder instance
          */
         public Builder withState(@NonNull String state) {
-            this.state = state;
+            this.values.put(KEY_STATE, state);
+            return this;
+        }
+
+        /**
+         * Use a custom nonce in the requests
```

Comment:
`Specify a custom nonce value to avoid replay attacks. It will be sent in the auth request that will be returned back as a claim in the id_token`